doc/release-notes/rl-2505.section.md

```diff
@@ -54,6 +54,12 @@
 - [testers.shellcheck](https://nixos.org/manual/nixpkgs/unstable/#tester-shellcheck) now warns when `name` is not provided.
   The `name` argument will become mandatory in a future release.
 
+- `grafana-agent` and `services.grafana-agent` have been removed in favor of
+  Grafana Alloy (`grafana-alloy` and `services.alloy`), as they depend on an EOL compiler version
+  and will become EOL during the 25.05 lifecycle.
+  Grafana recommends migrating to `grafana-alloy` (`services.alloy`).
+  See https://grafana.com/docs/alloy/latest/set-up/migrate/ for details.
```

Comment:
I forgot that `nix-build nixos/release.nix -A manual.x86_64-linux` wants URLs to be wrapped in `<>` in order to render them as links instead of literal text.

Other entries have the same issue, so this something for the release editors to fix before release.
Not worth another review cycle.

The eval errors below work as they should.